3-platform/warehouse/pom.xml

```
@@ -27,4 +27,24 @@
     <maven.compiler.target>17</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.6core.lib</groupId>
+      <artifactId>unit-test-bundle</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
```

get rid of `version` -> we are using `6core-bom` for this. 
move dependency to the app/domain module. 
replace all `junit`, `mockito` dependencies with your unit test bundle.
```suggestion
  <dependencies>
    <dependency>
      <groupId>com.6core.lib</groupId>
      <artifactId>unit-test-bundle</artifactId>
    </dependency>
  </dependencies>
```

---

Yes, this option is much better and simpler, thx